spec/controllers/target_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+require 'database_cleaner'
+
+DatabaseCleaner.strategy = :truncation
+
+describe TargetsController, type: controller do
+  before(:each) do
+    @target = create(:target)
+  end
+
+  describe 'POST create' do
+    it 'creates target' do
+      expect do
+        post :create, params: { target: @target.attributes }
+      end.to change(Target, :count).by(1)
+    end
+  end
+
+  describe 'GET index' do
+    it 'render a list of targets' do
+      DatabaseCleaner.clean
+      target1 = create(:target)
```

Comment:
you can do `targets = create_list(:target, 2)` and then match this array in the expectation